res/schema.xml

```diff
@@ -503,4 +503,31 @@ METADATA
       ALTER TABLE library ADD COLUMN coverart_digest BLOB;
     </sql>
   </revision>
+  <revision version="34" min_compatible="3">
+    <description>
+      Add last_played_at column to library table
+    </description>
+    <sql>
+      -- Add new column
+      ALTER TABLE library ADD COLUMN last_played_at DATETIME DEFAULT NULL;
+
+      -- Create missing indexes to speedup the following UPDATE statement
+      CREATE INDEX IF NOT EXISTS idx_PlaylistTracks_playlist_id_track_id ON PlaylistTracks (
+          playlist_id,
+          track_id
+      );
+      CREATE INDEX IF NOT EXISTS idx_PlaylistTracks_track_id ON PlaylistTracks (
+          track_id
+      );
+
+      -- Populate new column from history playlists
+      UPDATE library SET last_played_at=(
```

Comment:
This is done during the start-up window in Mixxx, without any safety net. 
Is there a risk?

Reply:
No. What safety net do you need other than a transaction??